nav2_costmap_2d/plugins/costmap_filters/binary_filter.cpp

```diff
@@ -97,8 +97,8 @@ void BinaryFilter::initializeFilter(
   base_ = BASE_DEFAULT;
   multiplier_ = MULTIPLIER_DEFAULT;
 
-  // Initialize state as "false" by-default
-  changeState(default_state_);
+  // Initialize state binary_state_ which at start its equal to default_state_
```

Comment:
If these are the same, why change it?

Reply:
'Initializing to binary_state_, which is equal to default_state_ on first bringup'

Reply:
So basically, this LGTM, no change needed